### ISSUE TITLE
When excluding docker image build step, the produced jar is of different size

### ISSUE THREAD

**Issue by philwinder** [mesos/elasticsearch#407]
When producing a jar file with `./gradlew build` a Jar with a size of 64MB. When appending the command with `-x buildDockerImage` to prevent the building of docker images, the jar file is a different size. Also it appears that the executor jar is not included in the final scheduler jar.

**Comment by philwinder** [comment]
It is a different size because the executor jar isn't copied into the scheduler when the buildDockerFile option is disabled. Figure out why this is.

**Comment by philwinder** [comment]
The copyJar task is not present when disabling buildDockerFile
```
:scheduler:compileJava
:scheduler:compileGroovy UP-TO-DATE
:scheduler:processResources
:scheduler:classes
:scheduler:jar
:scheduler:findMainClass
:scheduler:startScripts
:scheduler:distTar
:scheduler:distZip
:scheduler:bootRepackage
:scheduler:assemble
:scheduler:checkstyleMain
:scheduler:compileTestJava
```

**Comment by philwinder** [comment]
Fixed in 2a9b1201493522cf1cd0ccd1ace8f03a42641b7d.
